networkx/algorithms/dag.py

```diff
@@ -550,6 +610,29 @@ def transitive_closure(G, reflexive=False):
     NetworkXNotImplemented
         If `G` is not directed
 
+    Examples
```

Comment:
Just another organizational/narrative nit - it might be good to add a sentence to explain that these examples are intended to show the behavior of the `reflexive` kwarg, and to show the default behavior. e.g. something like:

```
The treatment of trivial (i.e. length 0) cycles is controlled by the `reflexive` parameter.
Trivial cycles do not create self-loops when ``reflexive=False`` (the default)::
   <example 2 here>
Trivial cycles do result in self-loops when ``reflexive=True``:
   <example 1 here>
A third option ...
```

Also, TBH I'm not particularly clear on the distinction between `reflexive=None` and `reflexive=False`. It might be good to add an example with the same input graph to show how the behavior differs.